packages/camera/src/components/Controls/__tests__/FullScreenButton.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react-native';
+import '@testing-library/jest-dom';
+import { I18nextProvider } from 'react-i18next';
+import i18n from '../../../i18n';
+import FullScreenButton from '../FullScreenButton';
+
+describe('Fullscreen Button component', () => {
+  it('renders the correct label', () => {
+    render(
```

Comment:
It's not good practice to use any provider/wrapper component in test cases. You can create a custom render method to render the React component. The method will take care of all wrappers required to a render a component in one place.